paasta_tools/config_utils.py

```diff
@@ -327,10 +332,25 @@ def merge_recommendations(
             log.debug(
                 f"Getting current configs for {service}/{instance_type_cluster}.yaml..."
             )
-            existing_configs = self.get_existing_configs(
-                service=service,
-                file_name=instance_type_cluster,
-            )
+            # i'm so sorry.
+            # basically, we need to make sure that for every autotuned service, we load both kubernetes-
+            # and eks- files for the existing configs, as there are services that at any given time will
+            # only exist on one of these or may have a mix (and the csv file that we get fakes the current
+            # cluster type)
```

Comment:
Do we think we need to add any extra note here that the eks-* file will 'win' in the current state since it's the 2nd dict, in case we have any instances with the exact same name appear in both files? (technically since this should only ever be during migration, they should have the exact same values, so maybe not terribly important to note? 🤞 )

Reply:
yea, it should only be during migration - but i can add a note

Reply:
```suggestion
            # only exist on one of these or may have a mix (and the csv file that we get fakes the current
            # cluster type)
            # NOTE: if an instance appears in both files, the counterpart will always "win" - this 
            # should only be possible while an instance is being migrated from one instance type to 
            # another
```